testdata/p4tc_samples_outputs/default_action_example_control_blocks.c

```diff
@@ -151,8 +147,8 @@ if (/* hdr->ipv4.isValid() */
                                 return TC_ACT_SHOT;
                         }
                     } else {
-                        return TC_ACT_SHOT;
-;
+/* drop_packet() */
```

Comment:
Indentation looks off in the c output

Reply:
This is coming from C code as well and indentation cant be added as we dont know while function processing how much indent to add, This is general function for MethodCallExpression